spring-cloud-gcp-samples/spring-cloud-gcp-sql-mysql-r2dbc-sample/pom.xml

```diff
@@ -35,6 +35,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-webflux</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.asyncer</groupId>
```

Comment:
Are you sure this is needed now? Why wasn't it needed with `dev.miku`?

Reply:
Removed - thanks!